translations/combat.tex/de.po

```diff
@@ -321,6 +466,9 @@ msgid ""
 "During these game modes, all enemy Units activate as described in the \\pagelink{AI Units}{AI Rules section}.\n"
 "\n"
 msgstr ""
+"\\subsection*{Kämpfe in Kampagnen- und kooperativen Szenarien}\n"
+"Abweichende Kampfregeln für diese Spiel\-modi finden sich unter \\pagelink{AI Units}{Bot-Regeln}.\n"
```

Comment:
I've looked in another PR i remembered. Why is it here no syntax-error? Because of the "{...}"?
![image](https://github.com/user-attachments/assets/52ae7ff5-2a3f-4930-b75b-7511dbd57791)